### PR TITLE
Autocomplete Revised

### PR DESCRIPTION
**Pull request by NiharikaVadlamudi**
Work for #19  
1. Resolved Tkinter * imports.
2.  Reason behind changing from 340 to 400: The boxes overlapped in the previous case ( Now I set it to 380, looks fine)
3. Changed variable name to make it more meaningful.

### CODE REVIEW

File: Attendance_app.py [hunk 1]
```diff
@@ -192,9 +196,21 @@ def __init__(self, parent, controller):
...
         )
         self.lb_class.place(x=170 * scale_factor, y=155 * scale_factor)
-        self.tv_class = tk.Entry(self, width=entry_width)
-        self.tv_class.focus()
-        self.tv_class.place(x=340 * scale_factor, y=160 * scale_factor)
+
+        # Class Codes Autocompletion Stuff
+        self.filled = False
+
+        self.classCodes =class_codes
```
Comment (Vibhu-Agarwal) [inline Attendance_app.py:203]: ⚠️ `E225 missing whitespace around operator.`
Forgot to apply code formatter?

⚠️ `camelCase usage for variable name`
Please use snake_case for variable names.

File: Attendance_app.py [hunk 2]
```diff
@@ -285,6 +301,31 @@ def exit(self):
         self.controller.destroy()
         self.controller.quit()
 
+    # Autocompletion Helper Functions
+    def match_string(self):
+        hits = []
+        classVar = self.class_variable.get()
```
Comment (Vibhu-Agarwal) [inline Attendance_app.py:307]: ⚠️ `camelCase usage for variable name`
Please use snake_case for variable names.

File: Attendance_app.py [hunk 3]
```diff
@@ -192,9 +196,21 @@ def __init__(self, parent, controller):
...
+        self.class_variable = StringVar(self)
+        self.class_variable.set("")  # default value
+        self.tv_class = tk.Entry(self, textvariable=self.class_variable)
+
+        self.tv_class.focus_set()
+        self.tv_class.bind("<KeyRelease>", self.get_typed)
+        self.tv_class.bind("<Key>", self.filled)
+        self.tv_class.icursor("end")
+
+        self.tv_class.place(x=380 * scale_factor, y=160 * scale_factor)
```
Comment (Vibhu-Agarwal) [inline Attendance_app.py:213]: 👉 The placement of text-boxes has got to be uniform on a screen.
If they are overlapping, it's a separate issue which you can raise on GitHub.
For now, keep in line with the rest of the text-boxes.

**Review by Vibhu-Agarwal — CHANGES_REQUESTED** [review]

**Comment by NiharikaVadlamudi** [comment]
@Vibhu-Agarwal Please review the code now.

**Review by Vibhu-Agarwal — APPROVED** [review]

**Pull request merged**